src/modules/deal-investor/dto/create-deal-investor-request.dto.ts

```diff
@@ -0,0 +1,67 @@
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { DateString, UkefId } from '@ukef/helpers';
+
+export type CreateDealInvestorRequest = CreateDealInvestorRequestItem[];
+
+export class CreateDealInvestorRequestItem {
+  @ValidatedStringApiProperty({
+    description: 'The unique identifier of the portfolio associated to the business entity. The UKEF portfolio is E1.',
+    example: 'E1',
+    minLength: 0,
+    maxLength: 2,
+    required: false,
+    default: 'E1',
+  })
+  readonly portfolioIdentifier: string;
+
+  @ValidatedStringApiProperty({
+    description: "The unique identifier for the deal within the portfolio. It will be a 10-digit code beginning with either '0020', '0030', or '0040'.",
+    example: '0020900111',
+    length: 10, // TO-DO: check
```

Comment:
Should we validate for this being exactly 10? Mulesoft does 0-10.

Reply:
Always 10 digits length. Mulesoft might accept letters and special characters, but this should be stopped. 

This is very common UKEF ID field, we could create custom decorator or validation type for it later.

Reply:
Cool, thanks Audrius, I've raised a separate ticket for this https://ukef-dtfs.atlassian.net/browse/APIM-172